VAMobile/src/api/payments/queryKeys.ts

```diff
@@ -0,0 +1,3 @@
+export const paymentKeys = {
+  payment: ['payment'] as const,
```

Comment:
```suggestion
  payments: ['payments'] as const,
```
Makes it a little more clear that this key is for a query responsible for getting multiple payments as opposed to just one